### PR TITLE
Updates for OCaml 5.3

### PR DESCRIPTION
**Pull request by vouillon**

### CODE REVIEW

File: compiler/lib/flow.ml [hunk 1]
```diff
@@ -416,7 +416,7 @@ let the_native_string_of ~target info x =
 let the_block_contents_of info x =
   match the_def_of info x with
   | Some (Block (_, a, _, _)) -> Some a
-  | Some (Prim (Extern "caml_make_array", [ x ])) -> (
+  | Some (Prim (Extern "caml_array_of_uniform_array", [ x ])) -> (
```
Comment (hhugo) [inline compiler/lib/flow.ml:419]: Don't you need both? Or normalize the primitive in the parser ?

Comment (vouillon) [reply compiler/lib/flow.ml:419]: I use `Primitive.alias` for that.
https://github.com/ocaml-wasm/wasm_of_ocaml/blob/c5e98de255d38f2fed15d1ad20d276bf9ae36463/compiler/lib-wasm/generate.ml#L1174

Comment (hhugo) [reply compiler/lib/flow.ml:420]: ok

Comment (hhugo) [reply compiler/lib/flow.ml:420]: Thinking more about, Are you sure this is enough? I would expect the alias to be resolved late (during code generation)

Comment (vouillon) [reply compiler/lib/flow.ml:420]: Indeed. I thought this was resolved much sooner.

**Pull request merged**